### DISCUSSION TITLE
graia图片部分的问题

### DISCUSSION THREAD

**Discussion by Spirild** [GraiaProject/Application#81] [Q&A]
情景：我用bot发送了一张本地的jpg图A，接收的master查看图是裂开的；我用master将A直接复制到对话框发回去（假设这份人工发的图实际为A‘），之前裂开的图一并被加载了出来。之后再发送同样的图A就都能加载出来。
检查：打断点检查过中间变量，二者的byte是一模一样的，图片ImageID也是一样的。此外不是所有的jpg图都存在这个问题。
想知道图裂的原因——是mirai框架的问题，bot发送的A post时少了什么信息，还是自己使用不当。
已知的问题：尚未确认A’附带的能被框架解析的json信息，是否和A的一致。如果仍然一致，是否有办法确认A‘与A实际的差别。

**Comment by GreyElaina** [comment]
更新到 mirai 最新版本...
